<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sonerit</class>
 <widget class="QMainWindow" name="sonerit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>1057</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sonerit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1260</width>
      <height>1010</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>680</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>STONERIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continue_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>750</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CONTINUE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1258</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
